modules/custom-status/custom-status.php

```diff
@@ -178,20 +124,16 @@ public static function register_custom_status_taxonomy(): void {
 	 * Makes the call to register_post_status to register the user's custom statuses.
 	 * Also unregisters pending, in case the user doesn't want them.
 	 */
-	public function register_custom_statuses() {
+	public static function register_custom_statuses(): void {
 		global $wp_post_statuses;
 
-			// Users can delete the pending status if they want, so let's get rid of that
-			// It'll get re-added if the user hasn't "deleted" them
-			unset( $wp_post_statuses['pending'] );
```

Comment:
I ended up removing this, and making the pending status as required as well. We assume right now that both drafts and pending review are going to be required, and won't be removed. If all statuses are removed, those two are given back. I also ran into a minor edge case where pending review was being created twice with different slugs. So by doing this, we are mirroring the behaviour that happens when no statuses exist. Based on what I saw in edit flow, those restrictions were put on purpose so this is better to do imo.

Reply:
I'd prefer we allow the "Pending Review" status to be removed. At present you're not even allowed to rename it which feels restrictive for some of the workflows we may expect that don't want to use that naming convention. I think it would be even better if we could also remove or rename "Draft" for flexibility, but if we require any statuses I think "Draft" is the only one broad enough to be it.

Reply:
https://github.com/Automattic/vip-workflow-plugin/pull/59#discussion_r1792705172